mobbex_subscriptions/controllers/front/notification.php

```diff
@@ -27,51 +26,8 @@ public function postProcess()
         // Get current action
         $action = Tools::getValue('action');
 
-        if ($action == 'callback') {
-            return $this->callback();
-        } else if ($action == 'webhook') {
+        if ($action == 'webhook')
```

Comment:
Si solo va a haber un action no seria mejor quitar ese param de la url y siempre que se pegue al endpoint llamar directamente tipo `$this->processWebhook()` ?